invenio_users_resources/templates/secutiry/email/new_user_via_admin.html

```diff
@@ -0,0 +1,5 @@
+<p>{{ _('Your new account has been created') }}</p>
```

Comment:
Typo in folder name `secutiry`, should this be `security` or Jinja won’t find the templates?